Source/App.js

```diff
@@ -1 +1,325 @@
+        url: 'https://cesiumjs.org/NewYork/3DTilesGml',
+        maximumScreenSpaceError: 16 // default value
+    }));
+
+    // Adjust the tileset height so its not floating above terrain
+    var heightOffset = -32;
+    city.readyPromise.then(function(tileset) {
+        // Position tileset
+        var boundingSphere = tileset.boundingSphere;
+        var cartographic = Cesium.Cartographic.fromCartesian(boundingSphere.center);
+        var surfacePosition = Cesium.Cartesian3.fromRadians(cartographic.longitude, cartographic.latitude, 0.0);
+        var offsetPosition = Cesium.Cartesian3.fromRadians(cartographic.longitude, cartographic.latitude, heightOffset);
+        var translation = Cesium.Cartesian3.subtract(offsetPosition, surfacePosition, new Cesium.Cartesian3());
+        tileset.modelMatrix = Cesium.Matrix4.fromTranslation(translation);
+    });
+
+    //////////////////////////////////////////////////////////////////////////
+    // Style 3D Tileset
+    //////////////////////////////////////////////////////////////////////////
+
+    var defaultStyle = new Cesium.Cesium3DTileStyle({
+        color : "color('white')",
+        show : true
+    });
+
+    city.readyPromise.then(function(tileset) {
+        tileset.style = defaultStyle;
+    });
+
+    var transparentStyle = new Cesium.Cesium3DTileStyle({
+        color : "color('white', 0.3)",
+        show : true
+    });
+
+    var heightStyle = new Cesium.Cesium3DTileStyle({
+        color : {
+            conditions : [
+                ["${height} >= 300", "rgba(45, 0, 75, 0.5)"],
+                ["${height} >= 200", "rgb(102, 71, 151)"],
+                ["${height} >= 100", "rgb(170, 162, 204)"],
+                ["${height} >= 50", "rgb(224, 226, 238)"],
+                ["${height} >= 25", "rgb(252, 230, 200)"],
+                ["${height} >= 10", "rgb(248, 176, 87)"],
+                ["${height} >= 5", "rgb(198, 106, 11)"],
+                ["true", "rgb(127, 59, 8)"]
+            ]
+        }
+    });
+
+    var tileStyle = document.getElementById('tileStyle');
+    function set3DTileStyle() {
+        var selectedStyle = tileStyle.options[tileStyle.selectedIndex].value;
+        if (selectedStyle === 'none') {
+            city.style = defaultStyle;
+        } else if (selectedStyle === 'height') {
+            city.style = heightStyle;
+        } else if (selectedStyle === 'transparent') {
+            city.style = transparentStyle;
+        }
+    }
+    tileStyle.addEventListener('change', set3DTileStyle);
+
+    //////////////////////////////////////////////////////////////////////////
+    // Custom mouse interaction for highlighting and selecting
+    //////////////////////////////////////////////////////////////////////////
+
+    // // If the mouse is over a point of interest, change the entity billboard scale and color
+    var previousPickedEntity = undefined;
+
+    var handler = new Cesium.ScreenSpaceEventHandler(viewer.scene.canvas);
```

Comment:
Though in general, I push users away from using SSE and towards using standard mouse move events.  It's important that we stress that using Cesium is similar to using any other browser widget/element.  There's nothing special about mouse events.